stellar-core-prometheus-exporter/debian/rules

```diff
@@ -3,25 +3,17 @@
 # output every command that modifies files on the build system.
 #export DH_VERBOSE = 1
 
+# See pybuild(1)
+export PYBUILD_NAME=stellar-account-prometheus-exporter
```

Comment:
should be `stellar-core-prometheus-exporter`

Reply:
Well spotted, fixed